### ISSUE TITLE
ext.pngmath causes UnicodeDecodeError

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#393]
I've been documenting a Python package with Sphinx. So far, I've been using jsmath extension, and it works fine. But I want to upload to packages.python.org, so I thought I should use pngmath instead. When I try switching to pngmath, I get:

```
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1456: ordinal not in range(128)
```

This is on Ubuntu, with Python 2.6 and Sphinx 0.6.5. The docs I'm trying to render are in the Hg repository at:

https://bitbucket.org/cmcqueen1975/cobs-python/

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/393
- Originally reported by: Craig McQueen
- Originally created at: 2010-04-26T14:04:56.174

**Comment by shimizukawa** [comment]
_From Craig McQueen on 2010-05-01 16:28:26+00:00_

I figured out that this only occurs if the math contains Unicode characters directly. I was using ≠. I changed to \ne and was able to build the documentation.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-05-22 12:34:33+00:00_

Thanks for the report; this should be fixed now in <<cset c41f516ddaa3b083e5aa5a5c982b2a7cc4819eb7>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:17+00:00_

Removing version: 0.6.5 (automated comment)
